python/nano/tutorial/learning rate search.ipynb

```diff
@@ -0,0 +1,253 @@
+   },
+   "source": [
+    "# Step2: Prepare data\n",
+    "We use fashion MNIST dataset for demonstration."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {
+     "base_uri": "https://localhost:8080/"
+    },
+    "id": "A4AjqFBTBfUp",
+    "outputId": "d80596f1-98e4-4358-a583-706e24c3da0c"
+   },
+   "outputs": [],
+   "source": [
+    "from tensorflow import keras\n",
+    "(x_train, y_train), (x_test, y_test) = keras.datasets.fashion_mnist.load_data()\n",
+    "\n",
+    "CLASSES = 10\n",
+    "\n",
+    "img_x, img_y = x_train.shape[1], x_train.shape[2]\n",
+    "x_train = x_train.reshape(-1, img_x, img_y,1).astype(\"float32\") / 255\n",
+    "x_test = x_test.reshape(-1, img_x, img_y,1).astype(\"float32\") / 255"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "hwrtVD9yBfUq"
+   },
+   "source": [
+    "# Step3: Build model\n",
+    "We now create our model. \n",
+    "\n",
+    "Decorate the model class with hpo.tfmodel, and you will be able to specify search spaces in init arguments when creating the model, as shown below. For more details, refer to [user doc](https://bigdl.readthedocs.io/en/latest/doc/Nano/QuickStart/hpo.html).\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "uRFIRllGr0xu"
+   },
+   "outputs": [],
+   "source": [
+    "import tensorflow as tf\n",
+    "from tensorflow.keras.datasets import mnist\n",
+    "from tensorflow.keras.layers import Conv2D, Dropout, MaxPooling2D\n",
+    "from tensorflow.keras.layers import Dense\n",
+    "from tensorflow.keras.layers import Flatten"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "SNmZHFMMBfUr"
+   },
+   "outputs": [],
+   "source": [
+    "import tensorflow as tf\n",
+    "@hpo.tfmodel()\n",
+    "class MyModel(tf.keras.Model):\n",
+    "\n",
+    "    def __init__(self, filters, kernel_size, strides, activation):\n",
+    "        super().__init__()\n",
+    "        self.conv1 = Conv2D(\n",
+    "            filters=filters,\n",
+    "            kernel_size=kernel_size,\n",
+    "            strides=strides,\n",
+    "            activation=activation)\n",
+    "        self.pool1 = MaxPooling2D(pool_size=2)\n",
+    "        self.drop1 = Dropout(0.3)\n",
+    "        self.flat = Flatten()\n",
+    "        self.dense1 = Dense(256, activation='relu')\n",
+    "        self.drop3 = Dropout(0.5)\n",
+    "        self.dense2 = Dense(CLASSES, activation=\"softmax\")\n",
+    "\n",
+    "    def call(self, inputs):\n",
+    "        x = self.conv1(inputs)\n",
+    "        x = self.pool1(x)\n",
+    "        x = self.drop1(x)\n",
+    "        x = self.flat(x)\n",
+    "        x = self.dense1(x)\n",
+    "        x = self.drop3(x)\n",
+    "        x = self.dense2(x)\n",
+    "        return x\n",
+    "model = MyModel(\n",
+    "    filters=hpo.space.Categorical(32, 64),\n",
```

Comment:
remove the search space in layers. Just use a normal keras model which don't have any search spaces.